src/app.ts

```diff
@@ -1,16 +1,30 @@
 import cors from '@koa/cors';
-import Koa from 'koa';
+import Router, { RouterContext } from '@koa/router';
+import Koa, { DefaultState, Middleware } from 'koa';
 import bodyParser from 'koa-bodyparser';
 import logger from 'koa-logger';
 import Articles from './articles';
 import apiDocumentation from './middleware/api-documentation';
 import errorHandler from './middleware/error-handler';
 import routing from './middleware/routing';
-import createRouter from './router';
 
-export default (articles: Articles): Koa => {
-  const app = new Koa();
-  const router = createRouter(articles);
+export type AppState = DefaultState;
+
+export type AppContext = RouterContext<AppState, {
```

Comment:
seems like a good alias

Reply:
does it matter that types are inside a top level module like `app`? Though they are used in conjunction with this module so it's probably the right place even if requiring this brings in everything.